onelive/index.html

```diff
@@ -94,12 +94,71 @@ <h1 class="display-3 text-center mb-5">Take a look at our past OneLives</h1>
                     <h1 class="display-3 text-center mb-5">Speakers</h1>
                 </div>
             </div>
-            <div class="row justify-content-center" id="btnSpeakers" style="visibility:hidden;">
+            <div  >
```

Comment:
Shall we remove the extra space on the `<div>` tag?

Reply:
Fixed